www/docs/api/index.php

```diff
@@ -142,27 +142,75 @@ function api_front_page($error = '') {
     if ($error) {
         print "<p style='color: #cc0000'>$error</p>";
     }
+
+$em = join('&#64;', array('enquiries', 'mysociety.org'));
+
 ?>
-<p>Welcome to TheyWorkForYou's API section, where you can learn how to query our database for information.</p>
 
 <h3>Overview</h3>
+<p>
+    Welcome to TheyWorkForYou's API section. The API (Application Programming
+    Interface) is a way of querying our database for information.
+</p>
+<p>
+    To use the API you need <a href="key">an API key</a>, and you may need a
+    license from us.
+</p>
+<p>
+    The documentation for each individual API function is linked from this
+    page: you can read what each function does, and test it out, without
+    needing an API key or to write any code.
+</p>
+
+<h3>Terms of usage</h3>
+<p>
+    Low volume, charitable use of the API is free. This means direct use by
+    registered charities, or individuals pursuing a non-profit project on an
+    unpaid basis, with a volume of up to 50,000 calls per year. All other use
+    requires a licence, as a contribution towards the costs of providing this
+    service. Please email us at <a href="mailto:<?=$em?>"><?=$em?></a> if your
+    usage is likely to fall into the non-free bracket: <u>unlicensed users
+    may be blocked without warning</u>.
+</p>
 
-<ol style="font-size:130%">
-<li>
-<?php if ($THEUSER->loggedin()) { ?>
-<a href="key">Get an API key (or view stats of existing keys)</a>.
-<?php } else { ?>
-<a href="key">Get an API key</a>.
-<?php } ?>
-<li>All requests are made by GETting a particular URL with a number of parameters. <em>key</em> is required;
-<em>output</em> is optional, and defaults to <kbd>js</kbd>.
-</ol>
+<h3>Pricing</h3>
+<p>
+    &pound;500 per year for up to 50,000 calls<br/>
+    &pound;1000 per year for up to 100,000 calls<br/>
```

Comment:
Very nitpicky, but given all other values are formatted with commas, should these be `1,000`, `2,000`, `2,500` and `3,000`?